components/brave_wallet/browser/brave_wallet_utils.cc

```diff
@@ -598,7 +598,8 @@ const mojom::NetworkInfo* GetZCashMainnet() {
   static base::NoDestructor<mojom::NetworkInfo> network_info(
       {chain_id,
        "ZCash Mainnet",
-       {""},  // TODO(cypt4): explorer url
+       {"https://zcashblockexplorer.com/transactions"},  // TODO(cypt4):
```

Comment:
nit: might be worth refactoring all of these urls into the constants file

Reply:
I'm fine with this being done in a follow up PR as well.

Reply:
I think we can move whole list of default network infos to another file